xcarchive/entitlements.go

```diff
@@ -0,0 +1,77 @@
+package xcarchive
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/env"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/go-xcode/plistutil"
+)
+
+func executableNameFromInfoPlist(infoPlist plistutil.PlistData) string {
+	if name, ok := infoPlist["CFBundleExecutable"].(string); ok {
+		return name
+	}
+	return ""
+}
+
+func getEntitlements(basePath, entitlementsRelativePath, executableRelativePath string) (plistutil.PlistData, error) {
+	entitlements, err := entitlementsFromXcentFile(basePath, entitlementsRelativePath)
```

Comment:
Do we still need the xcent file logic at all?

Reply:
I kept it because in our artefacts repo the MacOS archive still has it. I will double check it.

Reply:
I checked it and with the latest Xcode version (13 at the moment), macOS archives behave the same way, xcent file is missing from the archive.

I'm going to remove the xcent file logic.